WebRoot/public/login.jsp

```diff
@@ -93,7 +93,7 @@
 		</c:when>
 		<c:when test="${param.loginOp eq 'relogin' and not empty param.loginErrorCode}">
 			<zm:logout/>
-			<c:set var="errorCode" value="${param.loginErrorCode}"/>
+			<c:set var="errorCode" value="${zm:cook(param.loginErrorCode)}"/>
```

Comment:
same thing is done in hostedlogin.jsp also, see if we want to fix there as well

Reply:
Fixed.